manifests/consul.pp

```diff
@@ -4,52 +4,62 @@
 # This class uses solarkennedy/consul to initialize .
 #
 class libkv::consul(
-  $server = false,
-  $version = '0.8.5',
-  $use_puppet_pki = true,
-  $bootstrap = false,
-  $dont_copy_files = false,
-  $serverhost = undef,
-  $http_listen = '127.0.0.1',
-  $https_listen = '0.0.0.0',
-  $advertise = undef,
-  $datacenter = undef,
+  $firewall          = false,
```

Comment:
Tie this to the `simp_options` catalyst per the SIMP puppet-module-skeleton: https://github.com/simp/puppet-module-skeleton/blob/master/skeleton/manifests/init.pp.erb#L58